lib/acme/client.rb

```diff
@@ -101,13 +113,7 @@ def kid
 
   def new_order(identifiers:, not_before: nil, not_after: nil)
     payload = {}
-    payload['identifiers'] = if identifiers.is_a?(Hash)
-      identifiers
-    else
-      Array(identifiers).map do |identifier|
-        { type: 'dns', value: identifier }
-      end
-    end
+    payload['identifiers'] = self.class.prepare_order_identifiers(identifiers)
```

Comment:
Why not just use a private instance method instead of pushing it up has a class method?

Reply:
I agree it seems kind of wrong being a class method. Something in the setup for `client` sends a `GET https://127.0.0.1/directory` request and I wanted to avoid setting up VCR. Is there an existing cassette I can use or a way to skip the http request?

Reply:
IMO, it is just parameter handling. If you want to move it a private method and use `send(: prepare_order_identifiers )` like you would a public class method, I'm good with that.